### PR TITLE
dev-python/pyaudio: add support for Python 3.7 and 3.8

### PR DESCRIPTION
**Pull request by robert7k**
Closes: https://bugs.gentoo.org/719394

### CODE REVIEW

**Comment by robert7k** [comment]
@juippis I thought one shouldn't change a stable ebuild?

**Comment by juippis** [comment]
Python is different, because it's controlled through profiles/. We don't want to add extra keywording work for nothing.

**Comment by robert7k** [comment]
Done

File: dev-python/pyaudio/pyaudio-0.2.11.ebuild [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
```
Comment (juippis) [inline dev-python/pyaudio/pyaudio-0.2.11.ebuild:4]: Well, crap. I didn't realize there was an EAPI bump in it as well. You should mention it in the commit summary. 

So yeah... If you want EAPI-7 bump, **do** revbump and drop KEYWORDS. If you do, then also update the ebuild to use `distutils_enable_docs` function which should allow you to remove any `src_` phase from the ebuild altogether, along with `doc?` USE flags. 

I'm not sure if you can commit while editing EAPI-5 ebuild, but you can test it of course.

Comment (robert7k) [reply dev-python/pyaudio/pyaudio-0.2.11.ebuild]: Reverted that to keep it simple.

**Comment by juippis** [comment]
But you should become the maintainer and bump EAPI and take care of this ;)

**Pull request closed without merging**